zebra-chain/src/transaction.rs

```diff
@@ -144,6 +162,7 @@ impl Transaction {
             Transaction::V2 { .. } => None,
             Transaction::V3 { expiry_height, .. } => Some(*expiry_height),
             Transaction::V4 { expiry_height, .. } => Some(*expiry_height),
+            Transaction::V5 { expiry_height, .. } => Some(*expiry_height),
         }
     }
 
```

Comment:
We missed these changes:
* `sprout_nullifiers` should explicitly return empty for `V1` and `V5`
  * replace wildcard `_` with `V1 { .. } | V5 { .. }`
* `sapling_nullifiers` need a `V5` case
  * replace wildcard `_` with `V1 { .. } | V2 { .. } | V3 { .. }`

Let's also replace the wildcards, so we don't miss these changes for future transaction versions.

Reply:
Parsing Sapling in v5 transactions is now #1829.

Reply:
I added this at https://github.com/ZcashFoundation/zebra/pull/1824/commits/d541d2a0dfefb3000df4b1994499239b01a6b749 however we need to keep the wildcard. All the transaction versions are now covered explicitly in the 2 functions(`sprout_nullifiers` and `sapling_nullifiers`) but there are cases inside the versions that are not, so the default case is needed.

Reply:
I added suggestions for exhaustive matches.